docs/_static/swagger-ui.css

```diff
@@ -0,0 +1,3 @@
+.swagger-ui{
```

Comment:
These files(favicon-*.png index.html openapi.json swagger-*) can be generated by script dynamically. 
@zhaoqin-github could you please extract the _static file generation functions out of publish_api_ref.sh.